src/content/protocols/uniswap-v3.md

```diff
@@ -0,0 +1,117 @@
+| UniswapV3Factory                 | setOwner          | changes the owner to a new address                      | TimeLocked DAO contract (GovernorBravo) |
+| UniswapV3Factory                 | enableFeeAmount   | enables a fee amount for a given tick spacing           | TimeLocked DAO contract (GovernorBravo) |
+| UniswapV3Factory (UniswapV3Pool) | setFeeProtocol    | set the protocol's % share of the fees                  | TimeLocked DAO contract (GovernorBravo) |
+| UniswapV3Factory (UniswapV3Pool) | collectProtocol   | withdraw accumulated protocol fees to custom address    | TimeLocked DAO contract (GovernorBravo) |
+| ProxyAdmin                       | renounceOwnership | abandons ownership to the contract                      | TimeLocked DAO contract (GovernorBravo) |
+| ProxyAdmin                       | changeProxyAdmin  | changes ownership to the contract                       | TimeLocked DAO contract (GovernorBravo) |
+| ProxyAdmin                       | upgrade           | upgrades the contract                                   | TimeLocked DAO contract (GovernorBravo) |
+| TransparentUpgradeableProxy      | changeAdmin       | update the proxy admin                                  | TimeLocked DAO contract (GovernorBravo) |
+| TransparentUpgradeableProxy      | upgradeTo         | upgrade the NonfungibleTokenPositionDescriptor contract | TimeLocked DAO contract (GovernorBravo) |
+| TransparentUpgradeableProxy      | upgradeToAndCall  | upgrade the contract and call a function                | TimeLocked DAO contract (GovernorBravo) |
+| Ownable                          | transferOwnership | changes the owner                                       | 0x0                                     |
+| Ownable                          | renounceOwnership | abandons ownership                                      | 0x0                                     |
+| ProxyAdmin                       | transferOwnership | changes the owner                                       | 0x0                                     |
+| ProxyAdmin                       | renounceOwnership | abandons ownership                                      | 0x0                                     |
+
+## Dependencies
+
+No external dependency has been found.
+
+## Exit Window
+
+As the contracts are immutable the users can always withdraw their funds, but parameters such as protocol
+fees can be changed by the DAO. Timelocks protect the contracts and update are governed by the Governor Bravo contract.
+The lock period is at least two days and up to 30 days for governance actions.
+When a proposal is created (at least 2.5M Uni), the community can cast their votes during a 3 day voting period. If a majority, and at least 4M votes are cast for the proposal, it is queued in the Timelock, and may be executed in a minimum of 2 days.
```

Comment:
can you add one section at the end called `# Security Council`

and write: No security council needed because on-chain governance is in place.